deploy/bz1980755/10-bz1980755.CronJob.yaml

```
@@ -0,0 +1,74 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: bz1980755
+  namespace: openshift-sre-pruning
+spec:
+  failedJobsHistoryLimit: 3
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "25 3 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - preference:
+                  matchExpressions:
+                  - key: node-role.kubernetes.io/infra
+                    operator: Exists
+                weight: 1
+          tolerations:
+            - effect: NoSchedule
+              key: node-role.kubernetes.io/infra
+              operator: Exists
+          serviceAccountName: bz1980755
+          restartPolicy: Never
+          containers:
+          - name: bz1980755
+            image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+            imagePullPolicy: Always
+            args:
+            - /bin/bash
+            - -c
+            - |
+              set -e
+              set -o nounset
+              set -o pipefail
+
+              TEMPDIR=$(mktemp -d)
+              for ns in `oc get namespace -l 'hive.openshift.io/managed=true' -o jsonpath="{.items[*].metadata.name}"`; do
```

`configure-alertmanager-operator` is in `openshift-monitoring` namespace which does not have the hive label.  At best we could use the name of the namespace, all our operators in a NS that starts with "openshift"

```suggestion
              for ns in `oc get namespace -o jsonpath="{.items[*].metadata.name}" | grep "^openshift[-]" `; do
```

---

Great catch, thanks! I've addressed and tested this in the latest revision:

https://github.com/openshift/managed-cluster-config/blob/0c2f6d5b2e212d4f06f72defd5a935f60fc28c3f/deploy/bz1980755/10-bz1980755.CronJob.yaml#L43